app/controllers/runtime/stats_controller.rb

```diff
@@ -22,7 +22,7 @@ def stats(guid, opts={})
       end
 
       begin
-        stats = instances_reporters.stats_for_app(process)
+        stats, _ = instances_reporters.stats_for_app(process)
```

Comment:
Why do we ignore warnings in v2?

Reply:
> I don't quite understand why we ignore the errors in v2, although I'm guessing this is because v2 endpoints should not be modified and they don't support that warnings header?

Yeah, I didn't want to change v2 too much. It does have code in place that would support adding the header, though: https://github.com/cloudfoundry/cloud_controller_ng/blob/4a23a419862ef9b7272f411a4435e33fc7305f30/app/controllers/base/base_controller.rb#L124

Reply:
Sounds good to me!

Reply:
932e8ecac467ae656432a38f49a68b240c380a74